### PR TITLE
correct and complet CONST_vars

### PR DESCRIPTION
**Pull request by ochriste**
use correct folder for js auto discovery with ngeo, 
auto eval of closure_library_path variable

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -14,6 +14,16 @@ Version 2.0.0
...
 
+   Same for the auto evaluation of the `closure_library_path` configuration
+   variable:
+
+     node:
+        vars:
+        - closure_library_path
+        cmd: ["node", "-e"]
+
+   It is now directly defined in the CONST_vars.yaml file.
```
Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: Why does we need this ?

**Comment by ochriste** [comment]
because in Luxembourg it was originally included in the project yaml
https://github.com/Geoportail-Luxembourg/geoportailv3/blob/master/vars_geoportailv3.yaml

maybe this commit isnt clear, it means we need to **remove** these lines if they exist

**Comment by fredj** [comment]
@sbrunner PR updated

**Comment by sbrunner** [comment]
Looks good, merge :-)

**Pull request merged**